packages/scripts/src/admin-proposals/utils.js

```diff
@@ -187,7 +187,7 @@ const fundArbitrumParentMessengerForOneTransaction = async (web3Provider, from,
   console.log(
     `Arbitrum xchain messages require that the Arbitrum_ParentMessenger has at least a ${l1CallValue.toString()} ETH balance.`
   );
-  if (toBN(await web3Provider.eth.getBalance(arbitrumParentMessenger.options.address)) < toBN(l1CallValue)) {
+  if (toBN(await web3Provider.eth.getBalance(arbitrumParentMessenger.options.address)) < toBN(l1CallValue * 10)) {
```

Comment:
You also need to send `l1CallValue` * 10 ETH right? 

Also, the more I think about this, perhaps we should just set to 5, or some other config/CLI arg we can pass into the script...

With a default value of 2?

Reply:
If we did that, it would send 10X the amount required whenever it dropped to 9X the amount required, giving you 19X the amount required locked up. That might be a better UX experience for the proposer but would lock up more ETH. Not sure what trade-off is better there as a short-term fix.

And yeah, I think there's definitely cleaner solutions available when we do a broader refactoring of the scripts.

Reply:
I agree with @nicholaspai

I think we should send the difference between the threshold and the actual balance in the interest of correctness.

However, I think we should probably just bite the bullet and have this method take in the number of calls being made to arbitrum. This shouldn't be a major refactor, just a quick parameter change I would think.

Without doing either of these, I think modifying this check this way might make this script more difficult to use down the road rather than simpler.